modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -161,7 +161,7 @@
      * @param <U> the type of values held by the resulting {@code ObservableValue}
      * @param mapper the mapping function to apply to a value, cannot be {@code null}
      * @return an {@code ObservableValue} that holds the result of applying the given
-     *     mapping function on this {@code ObservableValue}, or {@code null} when it
+     *     mapping function on its value, or {@code null} when it
```

Comment:
I think "on this value", not "on its value", no?

Reply:
Yes, I fixed it.